components/raftstore-v2/src/operation/ready/mod.rs

```diff
@@ -63,7 +71,56 @@ use crate::{
     worker::tablet,
 };
 
-const PAUSE_FOR_RECOVERY_GAP: u64 = 128;
+const PAUSE_FOR_REPLAY_GAP: u64 = 128;
+
+pub struct ReplayWatch {
```

Comment:
how about changing skipped to normal_replay and paused to paused_for_replay?

Reply:
Sounds good to me.